### ISSUE TITLE
How can I reset current overlayed page

### ISSUE THREAD

**Issue by amosbird** [pdfpc/pdfpc#284]
Currently I have to navigate to the previous page and press `Space`.

**Comment by AndreasBilke** [comment]
I don't understand what you mean

**Comment by amosbird** [comment]
I hope this might explain what I meant https://la.wentropy.com/8wig.webm

**Comment by AndreasBilke** [comment]
No, I still don't get it. You are just showing me a video and not telling what keys you are hitting and what you want to achieve.

**Comment by amosbird** [comment]
Ok, let me rephrase the problem. I'd like to press one key to reset (clear) current page's overlay to the beginning state. So this ![](https://la.wentropy.com/A5sO.png) becomes this ![](https://la.wentropy.com/7iCh.png)

**Comment by AndreasBilke** [comment]
Okay now I understand what you want. But: IIRC we have not such an option
